cmd/build/environment.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/registry"
+)
+
+// SetServicetEnvVars set okteto build env vars
+func (*Command) SetServicetEnvVars(service, reference string) error {
```

Comment:
is the `t` after `service` intended? Also, I don'w see any case in which this function returns an error, I think we can delete the return type (unless it is needed for some interface). It would simplify the client as we wouldn't have to check the error